vectors_benchmark/src/predict.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
+//
+// nucliadb is offered under the AGPL v3.0 and as commercial software.
+// For commercial licensing, contact us at info@nuclia.com.
+//
+// AGPL:
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+//
+use reqwest::blocking::Client;
+use std::env;
+
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct PredictResults {
+    pub data: Vec<f32>,
+}
+
+/// Calls the predict service to convert the query as a vector set
+pub fn get_vectorset(query: &str, model: &str) -> PredictResults {
```

Comment:
I would avoid the term  `vectorset` here, as vectorsets is a nucliadb feature. What about `get_vector`? or `convert_query_to_vector`